distribution/src/main/gateway/bin/gateway.start

```diff
@@ -41,9 +41,17 @@ GW_HOME=`cd "$PRGDIR/.." ; pwd`
 
 # You can define various Java system properties by setting the value
 # of the GATEWAY_OPTS environment variable before calling this script.
-# The script itself should not be changed. For example, the setting
-# below sets the Java maximum memory to 512MB.
-[ -z "$GATEWAY_OPTS" ] && GATEWAY_OPTS="-Xmx512m"
+# The script itself should not be changed.
+if [ -z "$GATEWAY_OPTS" ]; then
+  # Sets the Java maximum memory to 512MB
+  GATEWAY_OPTS="-Xmx512m"
+  # Sets the java.security.edg to use /dev/urandom. By default /dev/random
+  # is used.  /dev/random is a blocking call which can hang on machines that
```

Comment:
Suggest:

By default, Java uses /dev/random to gather entropy data for cryptographic needs.  However, using /dev/random can cause delays during Gateway startup, especially in virtualized environments.  /dev/urandom does not require collection of entropy data in subsequent runs.  See:  https://github.com/kaazing/gateway/issues/167